cypress/integration/checkout/checkout.multipiece.items.spec.js

```diff
@@ -127,7 +127,7 @@ describe('Check Out', () => {
       { missingitemPieces : numberOfMissingPieces, missingDescription: missingPieces });
   };
 
-  it('C591 Check out: multipiece items (folijet) (vega)', { tags: [TestTypes.smoke, DevTeams.folijet] }, () => {
+  it('C591 Check out: multipiece items (folijet) (vega)', { tags: [TestTypes.smoke, DevTeams.folijet, DevTeams.vega] }, () => {
```

Comment:
Does itVega?

Reply:
yep, let's remove let's the tag for Folijet team

Reply:
Done